config/config.go

```diff
@@ -266,6 +279,39 @@ func readJsonConfigFromRepo(repoPath repos.RepoPath) (jsonConfig, error) {
 	return config, nil
 }
 
+// UnmarshalJSON implements the Unmarshaler interface for the jsonParents type, allowing the 'parent' field to be a single
+// struct or array of structs when defined in the config file.
+// @llr REQ-TRAQ-SWL-53
+func (op *jsonParents) UnmarshalJSON(data []byte) error {
+	if len(data) == 0 {
+		return errors.New("no bytes to unmarshal")
+	}
+
+	// Use the first character to determine the type
+	switch data[0] {
+	case '{':
+		var parent jsonParent
+		err := json.Unmarshal(data, &parent)
+		if err != nil {
+			return err
+		}
+		*op = append(*op, parent)
```

Comment:
Would it not make sense to plainly replace op altogether? Not sure what you'd be appending here to otherwise

Reply:
it seems it's easy enough with one, but not with a slice. So I left as append for consistency.